frb/data/Galaxies/180301/FRB180301_host.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Double check these match what is in the paper.

Reply:
Checked.